HW_06_Task1/HW_06_Task1/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace HW_06_Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите стихотворение, разделяя строки символом \";\"") ;
+            string txt = Console.ReadLine();
+            txt = Regex.Replace(txt,"а", "О",RegexOptions.IgnoreCase) ;
+            txt = Regex.Replace(txt, "a", "O", RegexOptions.IgnoreCase);
+            string[] array = txt.Split(new char[] { ';' });
+            for(int i=0;i<array.Length;i++)
```

Comment:
format issue `int i=0;i<array.Length;i++`

should be `int i=0; i < array.Length; i++`